Libraries/Hotcakes.Commerce.Dnn/packages.config

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="EntityFramework" version="6.1.1" targetFramework="net45" />
+  <package id="EntityFramework" version="6.4.4" targetFramework="net472" />
```

Comment:
What .NET version are the other projects running?  If we upgrade this one, we should upgrade them all, but only if it's required.  This update would overlap with and potentially break other features.  So, if the EF upgrade requires this, then we might need to update it everywhere now to avoid weird runtime issues later.

Reply:
All the projects are using the same version of .Net. With the EntityFramework update in the solutions that have direct reference to this, it has already been automatically updated in all the solutions. I have checked inside each solution in the /bin/Debug/ folder and the EntityFramework dll that appears in all of them is the dll with version 6.4.4. I've done some tests and haven't noticed any problems apparently.
1 - I have generated a new solution and installed it in a clean DNN solution and everything has worked correctly.
2 - I have configured the store with sample products and everything is correct too.
3 - I have carried out payment tests and I have not noticed any problems either.

Reply:
This pull request needs some changes before being approved.